les/costtracker.go

```diff
@@ -224,7 +264,7 @@ func (ct *costTracker) gfLoop() {
 						ct.gfLock.Unlock()
 						if ch != nil {
 							select {
-							case ct.totalRechargeCh <- uint64(ct.utilTarget * gf):
+							case ct.totalRechargeCh <- uint64(totalRecharge):
```

Comment:
we should broadcast the new `totalRecharge` to outside which is `ct.utilTarget * gf`

Reply:
Right, I fixed the logic.